tests/test_homogeneous_collections.py

```diff
@@ -121,10 +121,97 @@ class RootModel(BaseXmlModel, tag='model'):
     assert_xml_equal(actual_xml, xml)
 
 
+def test_text_list_extraction():
+    class RootModel(BaseXmlModel, tag="model"):
+        values: List[int]
+
+    xml = '''
+    <model>1 2 70 -34</model>
+    '''
+
+    actual_obj = RootModel.from_xml(xml)
+    expected_obj = RootModel(
+        values=[1, 2, 70, -34]
+    )
+
+    assert actual_obj == expected_obj
+
+    actual_xml = actual_obj.to_xml()
+    assert_xml_equal(actual_xml, xml)
+
+
+def test_text_tuple_extraction():
```

Comment:
Seems like this test is redundant because is tests the same case as the previous one.

Reply:
See comment below.